### ISSUE TITLE
Expiry Alert Per Purchase Products

### ISSUE THREAD

**Issue by BKB16** [uilibrary/Stocky-Issues-and-Feature-request#13]
Request features Expiry Alert Per Purchase Products

**Comment by naime-hossain** [comment]
Thank you for your Feature request, we will add it in our roadmap :) We will let you know when you add this feature, if you like our work, you can leave a review that will motivate our team toward further development if possible. If you don’t know how to provide ratings & review please visit this link: https://help.market.envato.com/hc/en-us/articles/203269490-Giving-Feedback-On-Items thank you

**Comment by kashifleo55** [comment]
Yes this is very important there should expiry date when add/edit product also should print on product label. Thanks
